### DISCUSSION TITLE
<Flex /> is missing some shorthand props

### DISCUSSION THREAD

**Discussion by viczam** [mantinedev/mantine#2883] [Feature requests]
The new `<Flex />` is awesome. Great addition to complete the already existing `<Stack />` and `<Group />` components.

But I noticed it's missing 3 props similar to the existing ones:
- basis
- grow
- shrink

Was there a specific reason why those props were left out?

**Comment by rtivital** [comment]
These styles are usually added to Flex component children and do not make sense on Flex component itself

**Reply by viczam** [reply]
it makes sense

**Reply by daddasoft** [reply]
do not make sense because this styles applied to flex box children not the parent maybe can be a component called ```Flex.Item``` that can revive this props and apply it to children

**Reply by daddasoft** [reply]
see this doc it's may help   https://www.w3schools.com/css/css3_flexbox_items.asp

**Reply by isnifer** [reply]
It absolutely makes sense. Flex component itself can be a child of a Flex component.
